src/components/SignalCard/SignalCard.module.scss

```diff
@@ -39,18 +39,21 @@
 }
 
 .title {
-  @include text-normal;
-
-  font-size: 18px;
   margin: 0;
   margin-bottom: 4px;
+  font-family: Rubik, sans-serif;
```

Comment:
@GarageInc We have mixin for it now in san-ui
@DmitriVanGuard Am I right?

Reply:
Exist mixin 'text-normal'. But I do't find mixin for 
font-style: normal;
  font-weight: 500;
  font-size: 22px;
  line-height: 30px;

Reply:
[`text('h4','m')` ?](https://github.com/santiment/san-ui/blob/master/src/mixins.scss)